src/AutoRest.CSharp/Common/Output/Expressions/Snippets.Linq.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Linq;
+using AutoRest.CSharp.Common.Output.Expressions.ValueExpressions;
+
+namespace AutoRest.CSharp.Common.Output.Models
+{
+    internal static partial class Snippets
+    {
+        public static class Linq
+        {
+            public static ValueExpression ToList(ValueExpression expression)
```

Comment:
since linq methods are all extension methods, I think we could define our expressions as extension methods here as well, and this should only applies to `EnumerableExpression`, therefore
```suggestion
            public static EnumerableExpression ToList(this EnumerableExpression expression)
```

Reply:
the `expression` being passed in here is a parameter reference which doesn't need to be enumerable.  Not sure we can make this change